CONTRIBUTING.md

```diff
@@ -1,55 +1,89 @@
-# Contributing to GraphQL PHP
+# Contributing to graphql-php
 
 ## Workflow
+
 If your contribution requires significant or breaking changes, or if you plan to propose a major new feature,
-we recommend you to create an issue on the [GitHub](https://github.com/webonyx/graphql-php/issues) with
-a brief proposal and discuss it with us first.
+we recommend you to [create an issue](https://github.com/webonyx/graphql-php/issues/new)
+with a brief proposal and discuss it with us first.
 
 For smaller contributions just use this workflow:
 
-* Fork the project.
-* Add your features and or bug fixes.
-* Add tests. Tests are important for us.
-* Check your changes using `composer check`.
-* Add an entry to the [Changelog's Unreleases section](CHANGELOG.md#unreleased).
-* Send a pull request.
+- Fork the project.
+- Add your features and or bug fixes.
+- Add tests. Tests are important for us.
+- Check your changes using `composer check`.
+- Add an entry to the [Changelog's Unreleased section](CHANGELOG.md#unreleased).
+- Send a pull request.
 
-## Setup the Development Environment
-First, copy the URL of your fork and `git clone` it to your local machine.
+## Setup
 
 ```sh
+git clone <your-fork>
```

Comment:
`your-fork-ssh-url` or sth like that?

Reply:
Some users might use HTTP. I guess URL covers both 👍